tools/sv-report

```diff
@@ -200,19 +201,22 @@ except KeyError as e:
 
 logger.info("Generating {} from log files in '{}'".format(args.out, args.logs))
 
+data = {}
+
 for tag in database:
     tag_usage[tag] = 0
 
-data = {}
 
-for r in [os.path.dirname(r) for r in glob(args.logs + "/*/")]:
-    r_name = os.path.basename(r)
-    logger.debug("Found Runner: " + r_name)
-    data[r_name] = {}
+def collect_logs(r_name):
+    my_data = {}
```

Comment:
Why `my_*` ? This does not seem to add more information to the user, or is it meant to indicate local-variableness ?

The `data` used to be the grab-bag of everything, now we can probably name it more specifically as `runner_data` ?

Reply:
Yes, I added it at first to easily distinguish what's in the thread and what's not, but you are right that the naming was poor. Changed the prefixes to `runner_` as you suggested.